package.json

```diff
@@ -40,6 +40,7 @@
     "compile:prod": "webpack --config node_modules/terra-dev-site/config/webpack/webpack.config.js -p",
     "danger": "danger ci",
     "deploy": "npm run compile:prod && gh-pages -d build",
+    "docs": "node scripts/generate-markdown/markdown.config.js",
```

Comment:
Lets rename this script something like `generate-package-readme` or `update-package-readme`

Reply:
Changes made: [86e2f87](https://github.com/cerner/terra-clinical/pull/445/commits/86e2f878d666e12dd8c5bc0abcdd050bc575585c)